pkg/providers/pricing/pricing.go

```diff
@@ -92,12 +95,14 @@ func NewAPI(sess *session.Session, region string) pricingiface.PricingAPI {
 	return pricing.New(sess, &aws.Config{Region: aws.String(pricingAPIRegion)})
 }
 
-func NewProvider(ctx context.Context, pricing pricingiface.PricingAPI, ec2Api ec2iface.EC2API, region string) *Provider {
+func NewProvider(ctx context.Context, pricing pricingiface.PricingAPI, ec2Api ec2iface.EC2API, instanceTypeCache *instancetype_cache.Provider, region string) *Provider {
```

Comment:
Is it possible to leverage the pricing controller without touching the instance types provider? I was thinking about these metrics and I was wondering if we could just use the raw data that we get from the pricing provider to hydrate these metrics.

Reply:
Do we need the vcpus and memory right now? Or can we just get the pricing information metric in and then come with the other info in a separate PR?